assets/scss/app.scss

```diff
@@ -90,13 +89,15 @@ header {
 }
 
 footer {
-  background: $rb-footer-background-color;
-  padding-top: 0.2em;
-  padding-bottom: 0.2em;
-  position: sticky;
```

Comment:
I was unable to use `sticky` position here to our purpose. It might be some bug in the browser itself related to using position: sticky with `bottom: 0` at mobile browser.